packages/ubuntu_desktop_installer/lib/pages/installation_complete/installation_complete.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+
+import '../../widgets.dart';
+import '../wizard_page.dart';
+import 'installation_complete_model.dart';
+
+const _kHighlightBackground = Color(0xFF0e8420);
+const _kHighlightForeground = Colors.white;
+
+class InstallationCompletePage extends StatelessWidget {
+  const InstallationCompletePage({Key? key}) : super(key: key);
+
+  static Widget create(BuildContext context) => InstallationCompletePage();
+
+  @override
+  Widget build(BuildContext context) {
+    final TextStyle? boldStyle = Theme.of(context)
+        .textTheme
+        .bodyText1!
+        .copyWith(fontWeight: FontWeight.bold);
+    return LocalizedView(
+      builder: (context, lang) => WizardPage(
+        title: Text(lang.installationCompleteTitle),
+        content: Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(top: 64, bottom: 32),
+              child: Image.asset(
+                'assets/version.png',
```

Comment:
I put image into assets, it is probably better to decide some path in the system and load it from file (for different versions)

Reply:
I think an asset is fine, at least for the time being.

Design commented on the issue (https://github.com/canonical/ubuntu-desktop-installer/issues/43#issuecomment-870672818) stating that it'd be great if the circular cropping could happen dynamically. Can you look into implementing that?